pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -262,6 +265,8 @@ func getRejectOFPorts(rejectType RejectType, sIface, dIface *interfacestore.Inte
 		inPort = uint32(sIface.OFPort)
 		outPort = uint32(dIface.OFPort)
 	case RejectServiceLocal:
+		fallthrough
```

Comment:
Could you add some comments here so that people with little context like me can understand this :P

Reply:
Sure. Added.